integration_tests/test_math.py

```diff
@@ -83,6 +83,34 @@ def test_lcm():
     i = lcm(21, -12)
     assert i == 84
 
+def test_log1p():
+    i: f64
+    i = 10.0
+    assert log1p(i) == log(i+1)
+
+def test_expm1():
+    i: f64
+    i = 2.0
+    assert expm1(i) == exp(i) - 1
```

Comment:
Hi, thanks for the PR. Just some suggestion here:
```suggestion
def test_expm1():
    i: f64
    eps: f64
    eps = 1e-12
    i = 2.0
    assert abs(expm1(i) - exp(i) + 1) < eps
```

Reply:
Hi @Smit-create, Thanks for the review!